calendar.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Community Calendar - Editing"
+author: "Graeme Stewart"
+layout: default
+---
+
+The HSF [Community Calendar](https://calendar.google.com/calendar/embed?src=e4v33e1a1drbncdle1n03ahpcs%40group.calendar.google.com&ctz=Europe/Amsterdam)
```

Comment:
May be remove the timezone in the URL?

Reply:
And after the `(link)`, may be add `{:target="_hsf_calendar"}` (no space between `()` and `{}`) to get the calendar in a new tab, being able to continue to read the doc...

Reply:
Fine be me, done for the next commit,